tests/release/pipelines/fbc_release.go

```diff
@@ -245,6 +246,9 @@ func createFBCReleasePlanAdmission(fbcRPAName string, managedFw framework.Framew
 			"fromIndex":            constants.FromIndex,
 			"targetIndex":          constants.TargetIndex,
 			"binaryImage":          constants.BinaryImage,
+			"publishingCredentials": strconv.Quote("fbc-preview-publishing-credentials"),
```

Comment:
why the `strconv.Quote`?

Reply:
yeah, we can probably just have normal strings, but otherwise lgtm

Reply:
Updated, thanks!